yarn.lock

```diff
@@ -1,3511 +1,5113 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
```

Comment:
Your Yarn lockfile was generated using Yarn v1. Just ensure it's v2+ for this to work